package.json

```diff
@@ -22,6 +23,7 @@
     "start:prefix": "gatsby build --prefix-paths && gatsby serve --prefix-paths",
     "dev": "gatsby develop",
     "dev:https": "gatsby develop --https --host localhost.corp.adobe.com --port 9000",
+    "dev:https:local": "gatsby develop --https --host localhost.corp.adobe.com --port 8000",
```

Comment:
Why is this needed?

Reply:
🤔 I didn't add this...?

Reply:
Woops, I've been looking at the wrong PR... 😓

Reply:
For local we use this port to run the application that's why I'm adding this. Okay I'll remove